install_config/topics/manually_configuring_for_gce-proc.adoc

```diff
@@ -0,0 +1,115 @@
+////
+Module included in the following assemblies:
+
+install_config/configuring_gce.adoc
+////
+
+= Manually configuring {product-title} for GCE
+
+=== Manually configuring master hosts for GCE
+
+
+.Procedure
+
+To configure {product-title} masters for GCE:
+
+. Edit the master configuration file at `/etc/origin/master/master-config.yaml` by
+default on all masters and update the contents of the `apiServerArguments` and
+`controllerArguments` sections:
++
+[source,yaml]
+----
+kubernetesMasterConfig:
+  ...
+  apiServerArguments:
+    cloud-provider:
+      - "gce"
+    cloud-config:
+      - "/etc/origin/cloudprovider/gce.conf"*
+  controllerArguments:
+    cloud-provider:
+      - "gce"
+    cloud-config:
+      - "/etc/origin/cloudprovider/gce.conf"*
+----
++
+[IMPORTANT]
+====
+When triggering a containerized installation, only the *_/etc/origin_* and
+*_/var/lib/origin_* directories are mounted to the master and node container.
+Therefore, ensure *_master-config.yaml_* is in the *_/etc/origin/master_*
+directory instead of *_/etc/_*.
+====
+
+. Create the `/etc/origin/cloudprovider/gce.conf` file with the previously
+mentioned content as:
```

Comment:
I'm not sure what the "previously mentioned content" is. Should they be entering info similar to that in the Ansible file when configuring using Ansible?